compiler/fir/raw-fir/fir-common/src/org/jetbrains/kotlin/fir/builder/BaseFirBuilder.kt

```diff
@@ -662,6 +660,7 @@ abstract class BaseFirBuilder<T>(val baseSession: FirSession, val context: Conte
         private fun generateComponentAccess(parameterSource: FirSourceElement?, firProperty: FirProperty) =
             buildQualifiedAccessExpression {
                 source = parameterSource
+                typeRef = firProperty.returnTypeRef
```

Comment:
@demiurg906 pointed out that here I missed to specify the type. Actually, it was a pure refactoring, and it's _not_ been set before.

Reply:
> I ran modularized test and number of untouched types came back to previous value

Good to hear this, and I assume this addition affects that.